<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupFrame</class>
 <widget class="QFrame" name="GroupFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTableWidget" name="table_realtime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>770</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="rowCount">
      <number>16</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>95</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>45</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="toolTip">
       <string extracomment="Device id">Device id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Target Power</string>
      </property>
      <property name="toolTip">
       <string extracomment="Target Power"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>R-T Power</string>
      </property>
      <property name="toolTip">
       <string extracomment="Real Time Power">Real Time Power</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Target Speed</string>
      </property>
      <property name="toolTip">
       <string extracomment="Target Spped"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>R-T Speed</string>
      </property>
      <property name="toolTip">
       <string extracomment="Real Time Speed">Real Time Speed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Running</string>
      </property>
      <property name="toolTip">
       <string extracomment="Running status">Running status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comm.</string>
      </property>
      <property name="toolTip">
       <string extracomment="Communication Status">Communication Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="toolTip">
       <string extracomment="Message"/>
      </property>
     </column>
     <item row="0" column="6">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Temperature </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumber_temperature"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_humidity">
          <property name="text">
           <string>Humidity</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="lcdNumber_humidity"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_group">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>62</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>62</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    background-image: url(:/skin/icons/dark/appbar.group.png);
 
    background-repeat:no-repeat;
    background-position: center bottom;
    background-color: transparent;
	border-style: none !important;
	border: none !important;
	border-radius: 5px;
    color: white;
    text-align: center bottom;
    padding-bottom: 2px;
	font-size:16px;
	font-weight: bold;

    background-color: rgb(14, 142, 231);
	border-style: outset;
}

QPushButton:hover{
    background-color:rgb(57, 159, 251);
    color: white;
    border: 0px solid rgba(14, 142, 231, 255);
}

QPushButton:checked{
    background-color: rgb(0, 125, 213);
    color: white;
    border: 0px solid rgba(14, 142, 231, 255);
}

QPushButton:flat{
    border: none;
}
QPushButton:pressed{
    background-color: rgb(0, 125, 213);
}

QPushButton:disabled{
    background-color: gray;
}

/*
QPushButton{
    background-color: rgb(14, 142, 231);
	border-style: outset;
    border-width: 2px;
	border-radius: 6px;
    color: white;
	font-size: 16px;
	font-weight: bold;
}

QPushButton:hover{
    background-color: rgb(57, 159, 251);
}



*/</string>
        </property>
        <property name="text">
         <string>ENTER</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>64</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
